src/test/java/com/rocana/lucene/codec/v1/TestBlockPostingsFormat2.java

```diff
@@ -32,8 +32,22 @@
 import org.apache.lucene.util.LuceneTestCase;
 import org.apache.lucene.util.TestUtil;
 
-/** 
- * Tests special cases of BlockPostingsFormat 
+/**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.TestBlockPostingsFormat2}.
+ *
+ * Why we forked:
+ *   - To use existing Lucene tests to test our {@link RocanaLucene50PostingsFormat}.
+ *
+ * What changed in the fork?
+ *   - Use {@link RocanaLucene50PostingsFormat} instead of Lucene's original.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * See ROCANA-8229
```

Comment:
Same comment here about referring to internal only JIRA tickets in open sourced code.